File: sudheesh001/fizzbuzz.cpp
#include <iostream>
using namespace std;
/* *******
   
   $ c++ fizzbuzz.cpp -o fizzbuzz
   $ ./fizzbuzz

   FizzBuzz:
   - Every number divisible by 3 and 5 display FizzBuzz
   - Every number divisible by 3 display Fizz
   - Every number divisible by 5 display Buzz

********** */

int main() {
   for (int i = 1; i <= 100; i++) {
       if (i % 3 == 0 && i % 5 == 0)
            cout << "FizzBuzz";
       else if (i % 3 == 0)
            cout << "Fizz";
       else if (i % 5 == 0)
            cout << "Buzz";
       else
            cout << i;
       cout << endl;
   }
   return 0;
}

File: aakash-cr7/fizzBuzz.js
function fizzBuzz(length = 100, fizzNumber = 3, buzzNumber = 5) {
    var output;
    for (var i = 1; i <= length; i++) {
        output = '';
        if (i % fizzNumber === 0) {
            output += 'Fizz';
        }
        if (i % buzzNumber === 0) {
            output += 'Buzz';
        }
        if (i % fizzNumber !== 0 && i % buzzNumber !== 0) {
            output += i;
        }
        console.log(output);
    }
}
